site/src/docs/getting-started/versioning.mdx

```diff
@@ -0,0 +1,60 @@
+---
+slug: '/getting-started/versioning'
+title: 'Versioning'
+---
+
+import ExternalLink from '../../components/ExternalLink';
+import LeadParagraph from '../../components/LeadParagraph';
+import InternalLink from '../../components/InternalLink';
+
+# Versioning in HDS
+
+<LeadParagraph>
+  HDS follows <ExternalLink href="https://semver.org/">Semver versioning</ExternalLink>. This page describes HDS
+  practices which versions are supported and what is considered as breaking changes.
+</LeadParagraph>
+
+## Which versions are supported?
+
+HDS supports current and one previous major version. Older versions are not supported anymore. We recommend updating to the latest version.
+
+## What is considered as a breaking change?
+
+Breaking change can occur either in design or in the implementation or in both.
+
+### Design
+
+These are breaking changes
+
+- if released component is removed completely
+- width of the released component changes (causing other elements possibly flowing to new row)
```

Comment:
Grammarly suggests something like this:
> width of the released component changes (causing other elements to  flow to a new row)

Should we mention height changes also?